### PR TITLE
3 implement navigation links with proper styling

### PR DESCRIPTION
**Pull request by Silverguilt**
Started styling.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1 +1,62 @@
+body {
+    font-family: 'Arial', sans-serif;
+    margin: 0;
+    padding: 0;
+  }
+  
+  header {
+    background-color: #333;
+    color: #FF5E00;
```
Comment (G-jacobsson) [inline css/styles.css:9]: I like this color scheme 😊

File: index.html [hunk 2]
```diff
@@ -9,6 +9,9 @@
         <link rel="stylesheet" href="/css/styles.css">
     </head>
     <body>
+
+<!--Header and navigation-->
+
```
Comment (G-jacobsson) [inline index.html:14]: Nice with some structure!

**Review by G-jacobsson — APPROVED** [review]
Looks good! 💪

File: css/styles.css (hunk 1 above)
Comment (Silverguilt) [reply css/styles.css:9]: Sweet!

**Pull request merged**